aszczachor/src/components/navigation/MobileMenu.js

```diff
@@ -0,0 +1,72 @@
+import styles from "components/navigation/mobileMenu.module.css";
+import { Link } from "react-router-dom";
+import { useState, useEffect, useRef } from "react";
+import ExpandButton from "./ExpandButton";
+
+const MobileMenu = ({
+  fashionStyling,
+  collections,
+  links,
+  openMenu,
+  menuBtn,
+}) => {
+  const [openFashionStyling, setOpenFashionStyling] = useState(false);
+  const [openCollections, setOpenCollections] = useState(false);
```

Comment:
skoro to zmienna typu boolean, wydaję mi się, że powinna się nazywać isCollectionOpened? poczytaj o tym: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig